llmfoundry/data/finetuning/dataloader.py

```diff
@@ -254,12 +259,23 @@ def build_finetuning_dataloader(
         ) if streams_cfg is not None else None
 
         # Take the constructor args from above, minus args that have been created separately
+        # dataset_constructor_args = {
+        #     k: v
+        #     for k, v in dataset_cfg.items()
+        #     if k in allowed_dataset_config_keys and k not in {
+        #         'streams',
+        #         'packing_ratio',
+        #         'auto_packing_replication',
+        #         'seq_parallel_replication',
+        #         'pad_to_longest',
+        #     }
+        # }
         dataset_constructor_args = {
             k: v
             for k, v in dataset_cfg.items()
-            if k in dataset_constructor_keys and
-            k not in {'streams', 'packing_ratio'}
+            if k in set(dataset_constructor_keys).union(extraneous_keys)
```

Comment:
pretty sure you still need to exclude streams and packing_ratio because they are passing in explicitly

Reply:
That's true. didn't see that happen.